### PR TITLE
Fixed finishedTest checks in run-workers command with rerun parameter

### PR DESCRIPTION
**Pull request by Ilrilan**
## Motivation/Description of the PR
- Description of this PR, which problem it solves
- Resolves #issueId (if applicable).

Applicable helpers:

- [ ] WebDriver
- [ ] Puppeteer
- [ ] Nightmare
- [ ] REST
- [ ] FileHelper
- [ ] Appium
- [ ] Protractor
- [ ] TestCafe
- [ ] Playwright

Applicable plugins:

- [ ] allure
- [ ] autoDelay
- [ ] autoLogin
- [ ] customLocator
- [ ] pauseOnFail
- [ ] puppeteerCoverage
- [ ] retryFailedStep
- [ ] screenshotOnFail
- [ ] selenoid
- [ ] stepByStepReport
- [ ] wdio

## Type of change

- [ ] :fire: Breaking changes
- [ ] :rocket: New functionality
- [x] :bug: Bug fix
- [ ] :clipboard: Documentation changes/updates
- [ ] :hotsprings: Hot fix
- [ ] :hammer: Markdown files fix - not related to source code
- [ ] :nail_care: Polish code

## Checklist:

- [ ] Tests have been added
- [ ] Documentation has been added (Run `npm run docs`)
- [ ] Lint checking (Run `npm run lint`)
- [ ] Local tests are passed (Run `npm test`)

### CODE REVIEW

File: lib/command/run-workers.js [hunk 1]
```diff
@@ -223,11 +229,21 @@ function simplifyObject(object) {
...
+        failures: -1,
+        tests: 0
+      };
+      appendStats(stats);
+    } else {
+      finishedTests[id].runs++
+    }
+  } else {
+    finishedTests[id] = test;
+    finishedTests[id].runs = 1
```
Comment (DavertMik) [inline lib/command/run-workers.js]: Codacy Issue found: [Missing semicolon.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5089116)

**Comment by DavertMik** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5089116&bid=16742208)

File: lib/command/run-workers.js [hunk 2]
```diff
@@ -46,6 +46,12 @@ module.exports = function (workers, options) {
     ...overrideConfigs,
   };
 
+  const configRerun = config.rerun || {};
+  let maxReruns = configRerun.maxReruns || 1;
+  if (maxReruns > 1) {
+    maxReruns = maxReruns * numberOfWorkers
```
Comment (DavertMik) [inline lib/command/run-workers.js]: Codacy Issue found: [Missing semicolon.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5089116)

Comment (DavertMik) [inline lib/command/run-workers.js]: Codacy Issue found: [Assignment can be replaced with operator assignment.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5089116)

File: lib/command/run-workers.js [hunk 3]
```diff
@@ -223,11 +229,21 @@ function simplifyObject(object) {
...
+const updateFinishedTests = (test, maxReruns) => {
   const { id } = test;
   if (finishedTests[id]) {
-    const stats = { passes: 0, failures: -1, tests: 0 };
-    appendStats(stats);
+    if (finishedTests[id].runs > maxReruns) {
+      const stats = {
+        passes: 0,
+        failures: -1,
+        tests: 0
```
Comment (DavertMik) [inline lib/command/run-workers.js]: Codacy Issue found: [Missing trailing comma.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5089116)

File: lib/command/run-workers.js [hunk 4]
```diff
@@ -223,11 +229,21 @@ function simplifyObject(object) {
...
+      };
+      appendStats(stats);
+    } else {
+      finishedTests[id].runs++
```
Comment (DavertMik) [inline lib/command/run-workers.js]: Codacy Issue found: [Missing semicolon.](https://app.codacy.com/manual/DavertMik/CodeceptJS/pullRequest?prid=5089116)

**Comment by DavertMik** [comment]
Use 

```
npm run lint-fix
```

**Comment by Ilrilan** [comment]
> Use
> 
> ```
> npm run lint-fix
> ```

Done

**Pull request merged**